6_extend_catalog.ipynb

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
<p>Could also <code>cd</code> using the absolute path? <code>/home/jovyan/work/cheapandfair-gateways-2024/heapandfair-template</code>  </p><p>That would also handle the odd case when someone has <code>cd</code>ed  to the work directory.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheapandfair/cheapandfair-gateways-2024/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I don't like being so tied to the tutorial configuraiton.</p><p>For example that won't work when I am developing on my laptop.</p>

Reply:
<p>the main purpose of this is to avoid issues if someone reruns the cell without restarting the notebook</p>